product-web-app/controller/capabilityController.ts

```diff
@@ -0,0 +1,19 @@
+import { Application, Request, Response } from 'express';
+import CapabilityService from '../service/capabilityService.js';
+import Capability from '../model/capability.js';
+
+export default class CapabilityController {
+  private capabilityServiceClass = new CapabilityService();
+
+  init(app: Application): void {
+    app.get('/capability-leads', async (req: Request, res: Response) => {
+      const capabilityService = this.capabilityServiceClass;
```

Comment:
it should be ok to just call this.capabilityServiceClass (name should be changed) instead of creating a const